server/main.py

```diff
@@ -7,16 +11,17 @@
 
 
 class Client:
-    """This class represents a client.
+    """A WebSocket client.
 
-    A client is identified by an `id` and contains the corresponding WebSocket that is
-    used to send and receive messages.
+    A client is identified by an ID and contains the corresponding WebSocket
+    that is used to send and receive messages.
     """
 
```

Comment:
Could you add this to the __init__ below. See https://github.com/Vthechamp22/kindly-kappa/pull/7/commits/2c56c617a19792e1b80a96c038a24842b6cf4613